deepspeed/runtime/comm/coalesced_collectives.py

```diff
@@ -41,9 +42,17 @@ def all_to_all_quant_reduce(tensors: List[Tensor], groups: {}) -> List[Tensor]:
     output_lst: List[Tensor] = [None] * len(tensors)
     for idx, tensor in enumerate(tensors):
         if tensor.dim() == 1:
-            intra_quant_group = global_world_size
+            logger.info(
```

Comment:
not sure whether this logger.info increase our all-to-all latency, If it increase our all to all time, please comment it out.